ray-operator/Dockerfile

```diff
@@ -1,5 +1,5 @@
 # Build the manager binary
-FROM registry.access.redhat.com/ubi9/go-toolset:1.20.10 as builder
```

Comment:
cc @anishasthana Would you mind providing more context on why we use UBI?

Reply:
gentlest of pings on this. Thanks in advance.